src/routes/index.ts

```diff
@@ -443,7 +443,13 @@ router.route(config.UPDATE_REVIEW_BENEFICIAL_OWNER_OTHER_URL)
   )
```

Comment:
should you have changes in this file? Might be leftover from UAR-151 before it was merged. I'd just undo all these changes unless you're adding a new page in your ticket

Reply:
This is to populate the service address fields on page load, although I appreciate this may be out of scope for my ticket. Can remove and take note for future fix?